shared/lsp/integration.test.ts

```diff
@@ -216,7 +216,7 @@ describe('register()', () => {
         assert.deepStrictEqual(selector, [
             {
                 language: 'typescript',
-                pattern: 'https://sourcegraph.test/repo@rev/-/raw/**',
+                pattern: 'https://sourcegraph.test/repo@rev/-/raw/#**/**',
```

Comment:
This looks wrong, neither the extension API nor the raw API uses these URLs?

Reply:
Oops, missed this one. I updated all other URLs in tests to `git://`, see below.

Reply:
Would you mind updating the tests? I'm not exactly sure where the mismatch is.

Reply:
https://github.com/sourcegraph/code-intel-extensions/pull/281